src/cape_ros_node.cpp

```diff
@@ -78,12 +78,17 @@ CapeRosNode::CapeRosNode(ros::NodeHandle nh)
 
   constexpr auto deg_to_rad = M_PI / 180;
 
-  auto max_angle = 6.0; // degrees
+  auto max_angle = 4.0; // degrees
   nh_.param("max_angle", max_angle, max_angle);
   cos_angle_max_ = std::cos(max_angle * deg_to_rad);
-  nh_.param("max_merge_distance", max_merge_dist_, max_merge_dist_);
+  nh_.param("max_merge_dist", max_merge_dist_, max_merge_dist_);
```

Comment:
Is the parameter name "max_merge_distance" or "max_merge_dist" ? Why the change?

Reply:
Just wanted to make it the same as max_merge_dist_